src/main/java/ogc/rs/database/DatabaseServiceImpl.java

```diff
@@ -74,18 +92,53 @@ public Future<List<JsonObject>> getCollections() {
                         " from collections_details join collection_supported_crs" +
                         " on collections_details.id = collection_supported_crs.collection_id join crs_to_srid" +
                         " on crs_to_srid.id = collection_supported_crs.crs_id join collection_type" +
-                        " on collections_details.id = collection_type.collection_id group by collections_details.id")
+                        " on collections_details.id = collection_type.collection_id where collection_type.type!= 'STAC' group by collections_details.id")
                     .collecting(collector)
                     .execute()
-                    .map(SqlResult::value))
+                    .map(SqlResult::value)
             .onSuccess(success -> {
-                LOGGER.debug("Collections Result: {}", success.toString());
-                result.complete(success);
+                if (success.isEmpty()) {
+                    LOGGER.error("Collections table is empty!");
+                    result.fail(
+                            new OgcException(404, "Not found", "Collection table is Empty!"));
```

Comment:
Cannot throw a `404` exception here because `/collections` is a valid path. If the table is empty just return an empty response or a JsonObject for `collections` object.